website/partials/status.html

```diff
@@ -0,0 +1,21 @@
+<div class="lh-1-5">
+  <h1 class="mb-50">Project Status</h1>
+
+  <h2>Is Protractor actively maintained?</h2>
+  <p>
+    Although active development on Protractor ended in December of 2022, the project will continue to be supported under <strong>Long Term Support</strong> and will continue to receive critical security &amp; compatibility fixes via the community through August 2023. This is important as there are many actively developed projects still benefiting from their existing code coverage, and we anticipate Protractor to continue functioning well into the future. 
+  </p>
+  <p>
+    In short: Protractor's current status is <strong>Long Term Support</strong> and <strong>will End Of Life in September 2023</strong>.
+  </p>
+  
+  <h2>What does End Of Life mean?</h2>
+  The packages will remain accessible on GitHub, npm, Bower, and Release archive. This website will also remain here indefinitely. The project will be archived on GitHub and remain accessible indefinitely, but will no longer be open to issues / comments / etc. 
+  
+  <h2>Support After August 2023</h2>
```

Comment:
Can we change the page to look like? https://docs.angularjs.org/misc/version-support-status

## Extended Long Term Support

If you need extended support for Protractor, you should consider:
- [herodevs.com](https://herodevs.com)

Reply:
@mgechev given that HeroDevs is supporting protractor for an additional year at no cost, it seems that it makes sense to explain the free year of NES here. I feel like that would make sense. Should we have a call to talk about this?